eZ/Publish/Core/REST/Common/Tests/FieldTypeProcessor/AuthorProcessorTest.php

```diff
@@ -0,0 +1,69 @@
+<?php
+
+/**
+ * File containing the DateProcessorTest class.
+ *
+ * @copyright Copyright (C) eZ Systems AS. All rights reserved.
+ * @license For full copyright and license information view LICENSE file distributed with this source code.
+ */
+namespace eZ\Publish\Core\REST\Common\Tests\FieldTypeProcessor;
+
+use eZ\Publish\Core\REST\Common\FieldTypeProcessor\AuthorProcessor;
+use PHPUnit\Framework\TestCase;
+
+class AuthorProcessorTest extends TestCase
+{
+    protected $constants = array(
+        'DEFAULT_CURRENT_USER',
+        'DEFAULT_EMPTY',
+    );
+
+    public function fieldSettingsHashes()
+    {
+        return array_map(
+            function ($constantName) {
+                return array(
+                    array('defaultAuthor' => $constantName),
```

Comment:
[]

Aka use `[]` in new code and lines that are changed, and leave `array` in places you don't change to unnecessary branch conflicts.